{
  "Header": {
    "Original": "CVE-2025-8869 (When extracting a tar archive pip may not check symbolic links point i ...)",
    "ID": "CVE-2025-8869",
    "Description": "(When extracting a tar archive pip may not check symbolic links point i ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-4348-1}",
      "Type": "xref",
      "Bugs": [
        "DLA-4348-1"
      ]
    },
    {
      "Original": "- python-pip 25.3+dfsg-1 (bug #1116336)",
      "Type": "package",
      "Package": "python-pip",
      "Kind": "fixed",
      "Version": "25.3+dfsg-1",
      "BugNo": 1116336
    },
    {
      "Original": "[trixie] - python-pip \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "trixie",
      "Package": "python-pip",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "[bookworm] - python-pip \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "bookworm",
      "Package": "python-pip",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://github.com/pypa/pip/pull/13550",
      "Type": "NOTE",
      "Description": "https://github.com/pypa/pip/pull/13550"
    },
    {
      "Original": "NOTE: Merge commit: https://github.com/pypa/pip/commit/f2b92314da012b9fffa36b3f3e67748a37ef464a (25.3)",
      "Type": "NOTE",
      "Description": "Merge commit: https://github.com/pypa/pip/commit/f2b92314da012b9fffa36b3f3e67748a37ef464a (25.3)"
    },
    {
      "Original": "NOTE: https://mail.python.org/archives/list/security-announce@python.org/thread/IF5A3GCJY3VH7BVHJKOWOJFKTW7VFQEN/",
      "Type": "NOTE",
      "Description": "https://mail.python.org/archives/list/security-announce@python.org/thread/IF5A3GCJY3VH7BVHJKOWOJFKTW7VFQEN/"
    }
  ]
}